Bluepill-cli/Bluepill-cli/Simulator/SimulatorMonitor.h

```diff
@@ -27,7 +25,9 @@ typedef NS_ENUM(NSInteger, SimulatorState) {
 
 @property (nonatomic, strong) SimDevice *device;
 @property (nonatomic, strong) NSString *hostBundleId;
-@property (nonatomic, assign) SimulatorState simulatorState;
+@property (nonatomic, assign) State appState;
```

Comment:
Add document how we decide the status and upon what circumstances, we update the statuses.

Reply:
Read the code.

Reply:
Prefer natural language to document the cases for future debugging.

Reply:
This is an internal monitoring state so it's only useful if you're actively working on the code, at which point you should just read the code. I don't see the point of documenting it outside the code but feel free to do it if you feel strongly about tit.

Reply:
`This is an internal monitoring state so it's only useful if you're actively working on the code` this doesn't make sense. Does `you should just read the code` mean that we don't need any documentation for anything except for public APIs?

I am ok with adding the documentation to the function impl that contains the logic.

Reply:
> I am ok with adding the documentation to the function impl that contains the logic.
Thank you! Please add it.